docs/faq.md

```diff
@@ -536,7 +536,9 @@ If the registry returns `401 Unauthorized` or `"code":"UNAUTHORIZED"`, it is oft
 * Different auth configurations exist in multiple places, and Jib is not picking up the auth information you are working on.
 * You configured a credential helper, but the helper is not on `$PATH`. This is especially common when running Jib inside IDE where the IDE binary is launched directly from an OS menu and does not have access to your shell's environment.
 * Configured credentials have access to the base image repository but not to the target image repository (or vice versa).
-* Typos in username, password, image names, or registry names.
+* Typos in username, password, image names, repository names, or registry names. (This is surprisingly common.)
```

Comment:
Don't think we need the parenthesis around `This is surprisingly common`. The use of the word surprisingly here has a strange feel to me as well? Perhaps, "This is a very common error."

Reply:
Ah, that sounds less colloquial and better.